docs/modules/index.rst

```diff
@@ -5,7 +5,8 @@
 .. toctree::
    :maxdepth: 2
 
+   optim
    averaging
+   dht
    client
-   server
-   dht
\ No newline at end of file
+   server
```

Comment:
Maybe sort alphabetically?

Reply:
Nope.
At least, I would like to see client and server next to each other.
Also, for now, optim and averaging seem most useful for an average user, thus we begin with them.